packages/grafana-ui/src/components/DeleteButton/DeleteButton.tsx

```diff
@@ -3,13 +3,19 @@ import React, { PureComponent, SyntheticEvent } from 'react';
 interface Props {
   onConfirm(): void;
   disabled?: boolean;
+  confirmText?: string;
```

Comment:
You make a change to this component but cannot śee that you actually use this component anywhere. Also is there a way to combine this component with the ConfirmButton? they seem very similar.

Reply:
I just started with DeleteButton first, but then created ConfirmButton. We can replace all usage of DeleteButton by ConfirmButton, but I prefer to make it in the separate PR.